lib/codegen/Cargo.toml

```diff
@@ -11,11 +11,13 @@ keywords = ["compile", "compiler", "jit"]
 build = "build.rs"
 
 [dependencies]
+cranelift-logger = { path = "../logger", version = "0.1.0", default-features = false }
 cranelift-entity = { path = "../entity", version = "0.16.1", default-features = false }
 failure = { version = "0.1.1", default-features = false, features = ["derive"] }
 failure_derive = { version = "0.1.1", default-features = false }
 hashmap_core = { version = "0.1.9", optional = true }
 target-lexicon = { version = "0.0.3", default-features = false }
+log = "0.4.3"
```

Comment:
Would it work to add `default-features = false` to the log dependencies (here and in the other Cargo.toml files)? That will allow us to continue to support `no_std` in these crates.